src/util.rs

```diff
@@ -124,7 +128,9 @@ pub unsafe fn pop_error(state: *mut ffi::lua_State, err_code: c_int) -> Error {
 
     if let Some(err) = get_wrapped_error(state, -1).as_ref() {
         ffi::lua_pop(state, 1);
-        err.clone()
+        // XXX: is this actually sound? we just popped it off the stack,
+        // didn't we?
+        err.0.clone()
```

Comment:
Lua runs GC only during `lua_load` / `lua_pcall` calls, so it's safe to pop value in this case.